routers/ask_gooey_new.py

```
@@ -0,0 +1,120 @@
+from daras_ai_v2.fastapi_tricks import get_route_path
+from daras_ai_v2.gooey_builder import (
+    can_launch_gooey_builder,
+    render_standalone_gooey_builder,
+)
+from daras_ai_v2.meta_content import raw_build_meta_tags
+from routers.custom_api_router import CustomAPIRouter
+from routers.root import get_og_url_path, page_wrapper
+from workspaces.models import Workspace
+
+META_TITLE = "Explore | Gooey.AI"
+META_DESCRIPTION = "Find, fork and run your field’s favorite AI recipes on Gooey.AI"
```

Ruff (0.16.0)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)